csshX

```diff
@@ -516,35 +516,33 @@ sub open_window {
     my $cmd = join ' ', map { s/(["'])/\\$1/g; "'$_'" } @args;
 
     # don't exec if debugging so we can see errors
-    unless ($config->debug) {
-      if (get_shell =~ /fish$/) {
-          $cmd = "clear; and exec $cmd" unless $config->debug;
-      } else {
-          $cmd = "clear && exec $cmd" unless $config->debug;
-      }
-    }
+    $cmd = "clear && exec $cmd" unless $config->debug;
```

Comment:
This looks like an unrelated change, and appears as though it will break compatibility with `fish` shells.

Also, turns the comment into a lie.